config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -1,78 +1,243 @@
-presubmits:
```

Comment:
moved the presubmit at the bottom as it's not going to change that often (same as master).